packages/babel-plugin-proposal-pipeline-operator/test/fixtures/fsharp/arrow-functions-parenless/exec.js

```diff
@@ -0,0 +1,34 @@
+const y = 2;
+
+const f = (x) => (x
+  |> (y) => y + 1)
+  |> (z) => z * y
+
+const _f = (x) => x
+  |> (y) => y + 1
+  |> (z) => z * y
+
+const g = (x) => x
+  |> (y) => (
+    y + 1
+    |> (z) => z * y
+  )
+
+const _g = (x) => x
+  |> (y => (
+    y + 1
+    |> (z) => z * y)
+  )
+
+const __g = (x) => x
+  |> (y => { return (
+    y + 1
+    |> (z) => z * y);
+    }
+  )
```

Comment:
All this random indentation makes it really hard to review this file :stuck_out_tongue:

Reply:
It is hard for me too :-p

I'm still getting the hang of how to indent pipelines. I'll see what I can do.

Reply:
Please see 4811f0861b860cde86de5c3c42fcdf6b8341b611 and let me know what you think.

Reply:
Yeah they look better. As a general rule, I think that if the pipeline body is multiline, `)` should aligned with the `|` of `|>`.

Let's hope Prettier implements them soon :stuck_out_tongue: